packages/mdc-grid-list/README.md

```diff
@@ -258,6 +258,7 @@ functions, with correct signatures:
 | Method Signature | Description |
 | --- | --- |
 | `getOffsetWidth() => number` | Get root element `mdc-grid-list` offsetWidth. |
+| `getNumberOfTiles() => number` | Get the number of `mdc-grid-tile`. |
```

Comment:
supernit: Just to make sure we're being crystal clear about what we need from implementors, could we change the description to "Get the number of `mdc-grid-tile` elements contained within the grid list"?